main.go

```diff
@@ -58,15 +62,14 @@ func main() {
 func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 
 	// Ignore all messages created by the bot itself
-	// This isn't required in this specific example but it's a good practice.
 	if m.Author.ID == s.State.User.ID {
```

Comment:
who is `m`?

Reply:
`m` stands for Message. I changed the param name to make it simpler.